Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -248,6 +248,15 @@
   - type: Sprite
     sprite: Clothing/Back/Backpacks/ertclown.rsi
 
+- type: entity
+  parent: ClothingBackpackERTLeader
+  id: ClothingBackpackERTChaplain
+  name: ERT chaplain backpack
+  description: A spacious backpack with lots of pockets, worn by Chaplains of an Emergency Response Team.
```

Comment:
Other descriptions you have "emergency response team" in all lower-case letters, but here you have written it in pascal case